### ISSUE TITLE
Intuition behind scaling LR by hvd.size() and backward_passes_per_step

### ISSUE THREAD

**Issue by amity137** [horovod/horovod#1539]
Hey folks, just wanted to know about the intuition behind multiplying the LR by hvd.size() and backward_passes_per_step. I know there was some discussion in the PR comments which enabled gradient accumulation, but seems like it wasn't complete or I didn't get it fully.

**Comment by tgaddair** [comment]
Hey @amity137, the main thing to keep in mind is that distributed training essentially increases the batch size you're using to train the model.  This makes since intuitively, since you're processing the same amount of data per epoch in fewer steps when you spread the work out among different workers.  As such, the batch size must be increasing as you do so.

Because the batch size has been increased, we need to increase the learning rate proportionately.  That would also hold true outside of the distributed training setting (double the batch size, double the learning rate).

With `backward_passes_per_step`, you're also increasing the batch size, just spreading the calculation over multiple forward passes in order to make better use of the GPU's bandwidth.  Two backward passes per step means two forward passes / batches for one update to the model.
